addons/website/static/src/less/website.edit_mode.less

```diff
@@ -44,9 +44,15 @@
 
     &.oe_structure.oe_empty, &[data-oe-type=html] {
         &:empty {
-            height: 300px;
-            background: url(/website/static/src/img/drag_here.png) center no-repeat;
-            background-size: 380px;
+            border: 2px dashed lighten(@gray-light, 20%);
+            margin: 20px;
+            padding: 112px 0px;
+            text-align: center;
+            &:before {
+                content: 'DRAG BUILDING BLOCKS HERE';
```

Comment:
As said on the task, this will not be translated so this is not valid. Solutions:

- If we really want to display text:
    - Use `content: attr(data-editor-message);` and use javascript to set the `data-editor-message` attribute on related elements
or
    - Extend the tip system (or create a new one) to display different kind of messages to the user
or
    - Do not use a `::before` element but use JS to display some text instead (but you have to handle that text removal when the structure is not empty anymore...)
- Use an image without any text or no image at all to replace that text

You should check with FGI.